sdk/storage/storage-blob/src/models.ts

```diff
@@ -319,3 +319,132 @@ export interface BlobQueryArrowField {
    */
   scale?: number;
 }
+
+/**
+ * A type that looks like a SAS permission.
+ * Used in {@link BlobSASPermissions} {@link ContainerSASPermissions} and {@link AccountSASPermissions} to parse SAS permissions from raw objects.
```

Comment:
Would better add comments in both places to remind maintainers to keep future permissions changes align.